.circleci/config.yml

```diff
@@ -15,8 +15,17 @@ jobs:
       - run: npm run lint
       - run: npm run build
       - run:
+          name: Test with go-ipfs
           command: npm run test
           no_output_timeout: 30m
+          environment:
+            IPFS_FLAVOR: go
+      - run:
+          name: Test with js-ipfs
+          command: npm run test
+          no_output_timeout: 30m
+          environment:
+            IPFS_FLAVOR: js
```

Comment:
Can we make these two tasks run in parallel?

Reply:
I guess we could create two different "jobs"?

Reply:
it works https://app.circleci.com/pipelines/github/ceramicnetwork/js-ceramic/4772/workflows/a56b2290-5fe5-4386-88bb-abb74552a6f4

Reply:
Nice!

Reply:
I wonder if there's a way to not duplicate the clone and build. With doing them in parallel we spend more CPU time but less wall clock time.

Reply:
In theory, you could do a single `build` job, and then run two ('with-go-ipfs', 'with-js-ipfs') jobs in parallel. In practice (prior experience+the internets on CircleCI specifics), with node packages, it gets slow in every dimension. Every CircleCI job executes in own container (or machine, or whatever executor). In order to pass data, like our fully installed and built repo, between jobs, you have to archive that, pass to storage, then download it and dearchive. For thousands of small files in node_modules, the process is far from perfect.

Reply:
ok, makes sense. Then let's keep as is :)